### DISCUSSION TITLE
Wallet is not recognized

### DISCUSSION THREAD

**Discussion by YannMarti** [WalletConnect/walletconnect-monorepo#4314] [Developer Support]
I have successfully registered my mobile wallet on the explorer and it shows up in the modal, but says the wallet is not installed even if it is. I figure i probably setuup something wrong on the WC project page, how exactly does WalletConnect check if thge wallet is installed on my phone?

**Comment by YannMarti** [comment]
Did some more tests and if i see this correctly, https://api.web3modal.com/getAndroidData will return wallet ids and corresponding android_app_id, which for my wallet seems to be null for some reason. Why would this be the case?

**Reply by glitch-txs** [reply]
hi, what's your project ID?

**Reply by YannMarti** [reply]
Its 8141668a92aad85f9f9f4b00e59914061df0f189b5b11fb35a15dc2708bb4596, the legacy wallet. I think i figured out the issue tho, have the wrong appstore link. Changes are waiting for review since over a week, but once its approved i can test if that was the oly issue, if not ill write again.
